server.js

```diff
@@ -834,7 +834,18 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
       var metricName = match[4];
       var format = match[5];
 
-      var sonarMetricName = metricName;
+      var options = {
+        headers: {
+          Accept: 'application/json'
```

Comment:
You can move the url into `options` too. Take a look at coveralls or codecov, e.g.